tut/landing.md

```diff
@@ -1,9 +1,12 @@
 This section of our documentation helps you learn the installation process of the Anbox Cloud Appliance and go through the first steps of using Anbox Cloud. Our tutorials aim to make as few assumptions as possible and provide a learning experience as you begin using Anbox Cloud.
 
-* [Install the Anbox Cloud Appliance on a dedicated machine](https://discourse.ubuntu.com/t/22681)
-* [Get started with Anbox Cloud (web dashboard)](https://discourse.ubuntu.com/t/24958)
-* [Get started with Anbox Cloud (CLI)](https://discourse.ubuntu.com/t/17756) 
-* [Set up a stream client](https://discourse.ubuntu.com/t/set-up-a-stream-client/37328)
-* [Create an addon](https://discourse.ubuntu.com/t/25284)
+The following tutorials help you install the Anbox cloud Appliance and start using it either through the web dashboard or through the command line interface:
+1. [Install the Anbox Cloud Appliance on a dedicated machine](https://discourse.ubuntu.com/t/22681)
+1. [Get started with Anbox Cloud using the web dashboard](https://discourse.ubuntu.com/t/24958)
+1. [Get started with Anbox Cloud using the command line interface](https://discourse.ubuntu.com/t/17756) 
+
+The following tutorials help you further explore features of Anbox Cloud:
+1. [Set up a stream client](https://discourse.ubuntu.com/t/set-up-a-stream-client/37328)
+1. [Create an addon](https://discourse.ubuntu.com/t/25284)
```

Comment:
Should we emphasize that these are optional?

Reply:
Reworded to note the same